src/psykal/mydomain_kern.f90

```diff
@@ -1,16 +1,28 @@
 module mydomain_kern_mod
 !
+use kind_params_mod, only : wp
+!
 implicit none
 !
+!type, extends(kernel_type) :: mydomain_type
+!  type(arg), dimension(2) :: meta_args = (/ &
+!    arg(WRITE, DIMS(3), CT), &
+!    arg(READ,  DIMS(3), CU, STENCIL(U)) /)
```

Comment:
I don't understand STENCIL(U). Does it mean 'Up'? If so, it should really be down as jk+1 is deeper than jk.